### ISSUE TITLE
AVT1: 4.1.2 Name, Role,Value-The 'id' downshift-1-label specified for the WAI-ARIA property 'aria-labelledby' value is not valid

### ISSUE THREAD

**Issue by pankaj-kumar34** [carbon-design-system/carbon#7513]
=====ENVIRONMENT=====
Browser: Chrome
DAP: with "12 May 2020 deployment"and "IBM Accessibility & IBM Accessibility Experimental" supported rulesets.

=====DEFECT DESCRIPTION=====
![image](https://user-images.githubusercontent.com/18532515/103850593-a6176180-50cd-11eb-9408-0943a6567a5c.png)
Check point: https://www.ibm.com/able/guidelines/ci162/name_role_value.html

it's dropdown component.
![image](https://user-images.githubusercontent.com/18532515/103850614-b596aa80-50cd-11eb-8ade-0f2d2a89b70f.png)

**Comment by sonabans** [comment]
Any outlook on the fix for this issue? It is blocking accessibility compliance for the offering.

**Comment by tw15egan** [comment]
I believe the a11y violation is because there is not a visible label, which is required. If you add a label, is the error still present?

**Comment by sonabans** [comment]
The faulty dropdown is the currency dropdown there (surlined in orange inn image below). There is a label here: "EUR". The problem there is that the Dropdown component does not seem to find it or understand that it is the selected item.
 

![image](https://user-images.githubusercontent.com/80099195/110301642-51855980-801e-11eb-8d38-b7c7030fccc6.png)

**Comment by tw15egan** [comment]
I believe it needs an actual label, confusingly named `titleText`, to prevent the error. If I do not provide that value, I am seeing the same error

**Comment by mateBe95** [comment]
seems to be not fixed still

**Comment by tw15egan** [comment]
cc @dakahn is a visible label a requirement for these elements? Or is there a way to make this accessible and hide the label text?

**Comment by dakahn** [comment]
That error is downshift attempting to programmatically link a label to it's corresponding aria. A visual label is required on all user inputs to meet [IBM Accessibility requirement 3.3.2](https://www.ibm.com/able/requirements/requirements/#3_3_2). 

Using placeholder text as a label is an anti-pattern that doesn't meet these requirements. See this [IBM Content Design guide for user input's](https://www.ibm.com/able/toolkit/design/content/user-input/#placeholders) for more details.

**Comment by tw15egan** [comment]
> A visual label is required on all user inputs to meet IBM Accessibility requirement 3.3.2.

Closing this as it is a valid a11y violation, and you will need to add a label to satisfy this requirement

**Comment by pankaj-kumar34** [comment]
@tw15egan, I don't see any option to provide aria-label for child component (dropdown values) in the Dropdown component. Can you please point me to any resource on how can we add or change the label?

**Comment by pankaj-kumar34** [comment]
`statuswise = [
    {
      id: 'datawise',
      text: `${i18n.t('monitor.label_datawise')}`,
    },
    {
      id: 'platformwise',
      text: `${i18n.t('monitor.label_platformwise')}`,
    }]

<Dropdown
            id="statuswise"
            type="inline"
            ariaLabel={i18n.t('monitor.label_statuswise')}
            label="statuswise"
             items={this.statuswise}
            }}
          />`
          
This is how we have added label but t's not working.

**Comment by tw15egan** [comment]
You can try setting the `titleText` prop, and then using `hideLabel` if you do not need a visibile label
